Frameworks/UIKit/UIImage.mm

```diff
@@ -123,6 +122,11 @@ + (UIImageCachedObject*)cacheImage:(UIImage*)image withName:(NSString*)name {
     return [obj autorelease];
 }
 
+static inline CGImageRef getImage(UIImage* self) {
```

Comment:
>self [](start = 43, length = 4)

please use a different name than self, it has special meaning and can get confusing.